TelemetryAPI/CosmosReader.cs

```diff
@@ -16,10 +18,8 @@ namespace TelemetryAPI
 {
     public static class CosmosReader
     {
-        static readonly DocumentClient _client = new DocumentClient(new Uri(Environment.GetEnvironmentVariable("CosmosDbUri")), Environment.GetEnvironmentVariable("CosmosDbAuthKey"));
-        static readonly FeedOptions feedOptions = new FeedOptions() { MaxItemCount = 30000, EnableCrossPartitionQuery = true };
-        static readonly string _collection = Environment.GetEnvironmentVariable("CosmosDataCollection") ?? "SimpleEvents";
-        static readonly string _database = Environment.GetEnvironmentVariable("CosmosDatabaseId") ?? "TelemetryDB";
+        static CosmosClient cosmosClient = new CosmosClient(Config.CosmosDbUri, Config.CosmosDbAuthKey);
```

Comment:
The naming convention for these fields lost the _ in front... its used elsewhere in the project.  

Can you either fix the other instances of _ in other files, or use _ in this case for consistency?

Reply:
fixed and consistent for both reader and sender